scss/_skeletons.scss

```diff
@@ -0,0 +1,60 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
+
+.skeleton-line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  // We use a &nbsp; to preserve
+  // the font size of the parent
+  &::before {
+    width: 0;
+    content: "\00A0";
+  }
```

Comment:
Even though I truly like this trick, wouldn't a grid-based layout be easier to use for anyone using Bootstrap?
Coouldn't we use this alongside `.row` and `.col`? Still thinking about it, but I like this way :)